clients/src/main/java/org/apache/kafka/common/serialization/Serdes.java

```diff
@@ -48,6 +59,11 @@
 
     static public final class DoubleSerde implements Serde<Double> {
         @Override
+        public void configure(Map<String, ?> configs, boolean isKey) {
+            // do nothing
```

Comment:
It's just noise. They can make sense in cases where it's not obvious that an empty implementation is desired, but `configure` is not one of those cases.